DevCycle.SDK.Server.Common/Model/Local/DVCRequestEvent.cs

```diff
@@ -1,45 +1,56 @@
 using System;
 using System.Collections.Generic;
 using System.Runtime.Serialization;
+using Newtonsoft.Json;
 
 namespace DevCycle.SDK.Server.Common.Model.Local
 {
     [DataContract(Name = "requestEvent")]
     public class DVCRequestEvent
     {
-        private const string VariableEvaluated = "variableEvaluated";
-        private const string VariableDefaulted = "variableDefaulted";
-
-        private readonly List<string> eventTypes = new List<string> {VariableEvaluated, VariableDefaulted};
+        private readonly List<string> eventTypes = new List<string>
+        {
+            EventTypes.aggVariableEvaluated,
+            EventTypes.variableEvaluated,
```

Comment:
Technically there isn't the concept of a single variable evaluated anymore for local.